catalyst/utils/config.py

```diff
@@ -17,6 +17,8 @@
 from catalyst import utils
 from catalyst.utils.tools.tensorboard import SummaryWriter
 
+from typing import Any, Dict, List, Union  # isort:skip
```

Comment:
Please follow catalyst import order described in the setup.cfg (https://github.com/catalyst-team/catalyst/blob/master/setup.cfg#L10). 

catalyst imports order:
```
from typing import ...  # isort:skip
import os  # import stdlibs
import ...

import numpy as pd # import core python libs
import pandas as np
...

import torch  # import dl libs
...

from catalyst import urils  # catalyst imports
...

Reply:
I have been removed `# isort:skip` and restarted `isort`

Reply:
Unfortunately, there is a bug in `isort` so it is not able to sort standard python libs in the custom order. Could you move `from typing import Any, Dict, List, Union  # isort:skip` line to the top of the file?

Reply:
fixed